src/node_modules/app/utils.js

```diff
@@ -1,5 +1,6 @@
 import * as _ from 'lamb';
 import {applyFnMap} from '@svizzle/utils';
+import {getESType} from 'app/elasticsearch';
 
```

Comment:
Unrelated to this PR, but I'd remove `capitalize` from this file and use `capitalise` from `src/node_modules/svizzle/utils/string-string.js` in `src/routes/builder/index.svelte`